### PR TITLE
Refactored the index route

### PR DESCRIPTION
**Pull request by ijlee2**
## Description

Through a series of small, meaningful commits, I made the following 4 changes:

- Refactored the `index` route to be more easily readable and maintainable
- Fixed 2 bugs (filter logic for merged pull requests didn't work before, and the repository name didn't show up before) 
- Updated the `github-pull` model to have correct attribute names
- Removed displaying pull requests made by a Bot

After changes:

after changes

## How to review

If you want to check all code changes, I recommend reviewing commits one at a time.

Otherwise, I recommend reviewing just 3 files:

- `app/models/github-pull.js`
- `app/routes/index.js`
- `app/utils/pull-request.js`

## What's next?

When I inspected the pull requests payload from GitHub, I saw the `user` value (an object) does _not_ contain the user's name. After we merge this PR, I think we will want to pursue the approach that was taken in #40.

Add unit tests for `index` route. Add application tests, possibly using `ember-cli-mirage`.

Clean up files (e.g. adapters and models) that we don't need. Remove packages that we don't need.

## References

- https://developer.github.com/v3/search/#search-issues-and-pull-requests

### CODE REVIEW

File: app/controllers/index.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+  @tracked newPRs = [];
+  @tracked mergedRFCs = [];
+  @tracked newRFCs = [];
+  @tracked contributorsList = '';
 
   @action
   async getContributors() {
-    const { mergedPulls } = this.model;
-    const fetchRequests = mergedPulls.map(pull => pull.get('user'));
+    const fetchRequests = this.mergedPRs.map(pullRequest => pullRequest.user);
```
Comment (ijlee2) [inline app/controllers/index.js:15]: If all we care about is the user's handle, we no longer need to make a fetch request. We can directly access the information via `pullRequest.user.login`.

But if we do want to get the user's name, we will want to continue to make the request.

**Comment by ijlee2** [comment]
Yep, please feel free to work on #40 next. Thanks!

**Comment by amyrlam** [comment]
@ijlee2 @abhilashlr first of all, really appreciate your help on this repo!

i think i didn't pull master last week

unfortunately this creates a regression, i'm running locally `git checkout ecfe035417f162bc82a210195f5d947e100f3dff`

image

i don't have time to look at the code right now, but i will try to soon.

**Comment by amyrlam** [comment]
isaac dm'd me. sorry i pasted in the wrong image ^^

**Pull request merged**